llvm/test/DebugInfo/X86/dbg-addr-dse.ll

```diff
@@ -46,8 +46,8 @@ entry:
 }
 
 ; ASM-LABEL: f: # @f
-; ASM: #DEBUG_VALUE: f:x <- [DW_OP_plus_uconst [[OFF_X:[0-9]+]]] [$rsp+0]
-; ASM: movl    %ecx, [[OFF_X]](%rsp)
+; ASM: movl    %ecx, [[OFF_X:[0-9]+]](%rsp)
+; ASM: #DEBUG_VALUE: f:x <- [DW_OP_plus_uconst [[OFF_X]]] [$rsp+0]
```

Comment:
@adrian-prantl do you have any concerns about this? I looked in the history and the code used to look like this. Just got re-ordered slightly.

Reply:
IIUC, the new order is more precise.